3478/1/Gemfile

```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org' do
+  gem 'dotenv'
+  gem 'nokogiri'
+  gem 'octokit'
```

Comment:
Read this: https://bundler.io/v1.3/rationale.html
It is better to lock versions of gems, so as not to catch problems with their updates.